wagtail/wagtailembeds/embeds.py

```diff
@@ -15,7 +15,11 @@ def get_embed(url, max_width=None, finder=None):
 
     # Get/Call finder
     if not finder:
-        finder = get_default_finder()
+        def finder(url, max_width=None):
+            for finder in get_finders():
+                if finder.accept(url):
+                    return finder.find_embed(url, max_width=max_width)
+
     embed_dict = finder(url, max_width)
 
```

Comment:
> On the flip-side of this: if a finder does accept a URL but subsequently throws EmbedNotFoundException, we won't proceed to the next finder in the list - in particular, this means that any finder listed after wagtail.wagtailembeds.finders.embedly will never take effect.

This is the expected and documented behaviour. If a finder is capable of retrieving an embed from a website but can't find it, it's unlikely that any other finder would have much luck. This prevents the module from making many HTTP requests (which block the server thread) to find embeds that don't exist.

If the wrong finder is being chosen due to misconfiguration, then the user should fix their configuration. If the user really does' want to check two finders (which I don't think is a common use case), they can write a custom finder class.